server/util.ts

```diff
@@ -0,0 +1,52 @@
+'use strict';
```

Comment:
@tillig Do we need this in a typescript file? the `tsconfig.json` file seems to already set `strict` (which should emit `use strict`)?

Reply:
Possibly not. I learned a lot about TypeScript as I was going and the `tsconfig.json` changed as I went. I probably added the thing in the `tsconfig.json` and just forgot to pull these out.

Reply:
Fixed - removed the explicit `use strict` from all .ts files.